superset/utils/machine_auth.py

```diff
@@ -23,6 +23,7 @@
 
 from flask import current_app, Flask, request, Response, session
 from flask_login import login_user
+from playwright.sync_api import BrowserContext
```

Comment:
you can move this import down to the authenticate_playwright_browser_context method if you get errors on it not loading when the feature flag is not on.

Reply:
I put the import behind a feature flag check - I couldn't move it to the method as it was used for typing that method

Reply:
Scratch that, importing feature_flag_manager causes a circular import